api/src/pcapi/core/finance/api.py

```diff
@@ -1564,16 +1575,83 @@ def _prepare_invoice_context(invoice: models.Invoice) -> dict:
             lines=lines,  # type: ignore [arg-type]
         )
         groups.append(invoice_group)
+    reimbursements_by_venue = get_reimbursements_by_venue(invoice)
 
     venue = offerers_repository.find_venue_by_siret(invoice.businessUnit.siret)  # type: ignore [arg-type]
+    period_start, period_end = get_invoice_period(invoice.date)
     return dict(
         invoice=invoice,
         groups=groups,
         venue=venue,
         total_used_bookings_amount=total_used_bookings_amount,
         total_contribution_amount=total_contribution_amount,
         total_reimbursed_amount=total_reimbursed_amount,
+        period_start=period_start,
+        period_end=period_end,
+        reimbursments_by_venue=reimbursements_by_venue,
+    )
+
+
+def get_reimbursements_by_venue(invoice: models.Invoice) -> typing.List[typing.Dict[str, str]]:
+    reimbursments_by_venue = []
+    common_columns = (
+        offerers_models.Venue.id.label("venue_id"),
+        offerers_models.Venue.publicName,
+    )
+
+    pricing_query = (
+        models.Invoice.query.join(models.Invoice.cashflows)
+        .join(models.Cashflow.pricings)
+        .filter(models.Invoice.id == invoice.id)
+    )
+
+    query = (
+        pricing_query.with_entities(
+            *common_columns,
+            models.Pricing.amount.label("pricing_amount"),
+            bookings_models.Booking.amount.label("booking_amount"),
+            bookings_models.Booking.individualBookingId,
+        )
+        .join(models.Pricing.booking)
+        .join(bookings_models.Booking.venue)
+        .order_by(offerers_models.Venue.id, offerers_models.Venue.publicName)
```

Comment:
idem plus haut, je pense qu'il faudrait : 

```suggestion
        .order_by(offerers_models.Venue.id, coalesce(offerers_models.Venue.publicName, offerers_models.Venue.name))
```